manifests/pulp.pp

```diff
@@ -84,18 +84,41 @@
   Boolean $manage_mongodb = true,
   String $pub_dir_options = '+FollowSymLinks +Indexes',
   Stdlib::Absolutepath $repo_export_dir = '/var/lib/pulp/katello-export',
+  Boolean $manage_httpd = false,
+  Optional[Stdlib::Absolutepath] $https_cert = undef,
```

Comment:
I did intentionally use `undef` as default to not unintentionally break things. In the default setup this would be identical as before.
The default of $https_cert in the Pulp module is "/etc/pki/pulp/ca.crt" which is different from `certs::apache`, or did I miss something?

Reply:
In puppet-foreman_proxy_content we use the CA cert:
https://github.com/theforeman/puppet-foreman_proxy_content/blob/47e9711cb38a1da8405adc16426b802f4c76021e/manifests/init.pp#L272

I'm not sure that's correct though. @ehelms thoughts?